tests/basic_test.go

```diff
@@ -20,12 +20,12 @@ func TestSparkOperatorInstallation(t *testing.T) {
 	defer spark.CleanUp()
 
 	if err != nil {
-		t.Error(err.Error())
+		t.Fatal(err.Error())
 	}
 
 	k8sNamespace, err := spark.Clients.CoreV1().Namespaces().Get(spark.Namespace, v1.GetOptions{})
 	if err != nil {
-		t.Error(err.Error())
+		t.Fatal(err.Error())
```

Comment:
This is just to stop test execution and prevent logging a message about successful operator installation to be printed, since it's may be not true depending on the error.